lib/dal/src/attribute/value.rs

```diff
@@ -2193,9 +2171,9 @@ impl AttributeValue {
     pub async fn prop(
         ctx: &DalContext,
         attribute_value_id: AttributeValueId,
-    ) -> AttributeValueResult<Prop> {
+    ) -> AttributeValueResult<PropNodeWeight> {
```

Comment:
Don't know that this needs to change right now, but changing this to return a `PropNodeWeight` feels "leaky"; like it's exposing information that shouldn't make it out this far. Especially in a `pub` function. If it weren't `pub` I don't think it would be as much of a concern.

Reply:
I think it's important that none of the callers use the CAS parts of Prop, though, and I'm sort of loathe to revert it so they are grabbing it ... would it be worth changing all the callers to do `Prop::node_weight(ctx, AttributeValueId::prop_id(ctx, av_id).await?).await?`?

Maybe `AttributeValue::prop_node()` or something.